probes/hasNoGitHubWorkflowPermissionUnknown/def.yml

```diff
@@ -24,12 +24,12 @@ outcome:
 remediation:
   effort: Low
   text:
-    - Visit https://app.stepsecurity.io/secureworkflow/${{ metadata.repo }}/${{ metadata.workflow }}/${{ metadata.branch }}?enable=permissions
+    - Visit https://app.stepsecurity.io/secureworkflow/${{ metadata.repository.uri }}/${{ metadata.workflow }}/${{ metadata.branch }}?enable=permissions
     - Tick the 'Restrict permissions for GITHUB_TOKEN'
     - Untick other options
     - "NOTE: If you want to resolve multiple issues at once, you can visit https://app.stepsecurity.io/securerepo instead."
   markdown:
-    - Visit [https://app.stepsecurity.io/secureworkflow](https://app.stepsecurity.io/secureworkflow/${{ metadata.repo }}/${{ metadata.workflow }}/${{ metadata.branch }}?enable=permissions).
+    - Visit [https://app.stepsecurity.io/secureworkflow](https://app.stepsecurity.io/secureworkflow/${{ metadata.repository.uri }}/${{ metadata.workflow }}/${{ metadata.branch }}?enable=permissions).
```

Comment:
I think the branch template also needs to change to  `repository.defaultBranch`?

Reply:
https://github.com/ossf/scorecard/pull/3816/commits/c56806d8aa5032991974981a098e5c489cef807a